addons/mail/static/tests/new/views/chatter_topbar_tests.js

```diff
@@ -180,3 +181,85 @@ QUnit.test("log note/send message switching", async function (assert) {
         ".o-mail-composer .o-mail-composer-textarea[placeholder='Log an internal note...']"
     );
 });
+
+QUnit.test("attachment counter without attachments", async function (assert) {
+    const pyEnv = await startServer();
+    const resPartnerId = pyEnv["res.partner"].create({});
+    const { openView } = await start();
+    await openView({
+        res_id: resPartnerId,
+        res_model: "res.partner",
+        views: [[false, "form"]],
+    });
+    assert.containsOnce(target, ".o-mail-chatter-topbar-add-attachments");
+    assert.containsOnce(target, ".o-mail-chatter-topbar-add-attachments:contains(0)");
```

Comment:
Wouldn't it make more sense to assert `.o-mail-chatter-topbar-add-attachments` text is 0 rather than checking there is an element in the dom that contains said text?

Reply:
It's relatively the same: `:contains()` is a bit more flexible because you don't need to target the exact children, like the `span`. And it's also less accessors as you can put it in query. The shape of assertions are the same, so it's faster to grasp what the test does at a glance.